cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java

```diff
@@ -2,54 +2,73 @@
 
 
 import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.cristin.common.handler.CristinQueryHandler;
 import no.unit.nva.model.Organization;
-import nva.commons.apigateway.ApiGatewayHandler;
 import nva.commons.apigateway.RequestInfo;
 import nva.commons.apigateway.exceptions.ApiGatewayException;
 import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.apigateway.exceptions.NotFoundException;
+import nva.commons.core.Environment;
 import nva.commons.core.JacocoGenerated;
 import nva.commons.core.paths.UriWrapper;
 
 import java.net.HttpURLConnection;
 import java.net.URI;
-import java.util.Optional;
 import java.util.regex.Pattern;
 
 import static no.unit.nva.cristin.common.ErrorMessages.ERROR_MESSAGE_BACKEND_FAILED_WITH_STATUSCODE;
 import static no.unit.nva.cristin.common.ErrorMessages.ERROR_MESSAGE_INVALID_PATH_PARAMETER_FOR_ID_FOUR_NUMBERS;
 import static no.unit.nva.cristin.model.Constants.BASE_PATH;
+import static no.unit.nva.cristin.model.Constants.CRISTIN_API_BASE;
 import static no.unit.nva.cristin.model.Constants.DOMAIN_NAME;
 import static no.unit.nva.cristin.model.Constants.HTTPS;
+import static no.unit.nva.cristin.model.Constants.NOT_FOUND_MESSAGE_TEMPLATE;
 import static no.unit.nva.cristin.model.Constants.ORGANIZATION_PATH;
+import static no.unit.nva.cristin.model.Constants.UNITS_PATH;
 import static no.unit.nva.cristin.model.JsonPropertyNames.IDENTIFIER;
+import static no.unit.nva.model.Organization.ORGANIZATION_CONTEXT;
+import static no.unit.nva.model.Organization.ORGANIZATION_IDENTIFIER_PATTERN;
 
-public class FetchCristinOrganizationHandler extends ApiGatewayHandler<Void, Organization> {
+public class FetchCristinOrganizationHandler extends CristinQueryHandler<Void, Organization> {
```

Comment:
`CristinQueryHandler` is meant for queries, not fetch. Im working on this in another branch, where i can reuse a lot of things from `CristinHandler` and `CristinQueryHandler` to reduce duplicate code

Reply:
I see you have extended it in your Query class as well. This reduces the scope of my other branch, as we have done exactly the same. Maybe I dont need to do this refactoring in a separate PR but include it in the one im currently working on as there now is less lines of code in my branch to merge

Reply:
It'is prepared for the furure and APIGatewayHandler is grandfather anyway...